.github/actions/build/action.yml

```
@@ -42,7 +42,7 @@ runs:
       run: |
         mvn --batch-mode --update-snapshots clean compile test-compile
         mvn --batch-mode package -DskipTests -Dmaven.test.skip=true
-        mvn --batch-mode install -DskipTests -Dmaven.test.skip=true -Dgpg.skip
+        mvn --batch-mode install -DskipTests -Dmaven.test.skip=true -Dgpg.skip -Dspotless.check.skip
```

Let's realign the arguments:
```suggestion
        mvn --batch-mode -DskipTests -Dmaven.test.skip=true -Dgpg.skip -Dspotless.check.skip install
```